packages/clerk-js/src/ui/elements/Select.tsx

```diff
@@ -85,38 +94,51 @@ export const Select = <O,>(props: PropsWithChildren<SelectProps<O>>) => {
     [togglePopover, onChange],
   );
 
+  let newChildren = children;
+  if (React.Children.count(children) === 0) {
+    newChildren = (
+      <>
+        <SelectOptionList />
+        <SelectButton />
+      </>
+    );
+  }
+
```

Comment:
This is me nitpicking, and there's no need to change anything. I'd write this as:
```tsx
    const defaultSelectList = (
      <>
        <SelectOptionList />
        <SelectButton />
      </>
    );


// ...

{ !!React.children.count(children) ? children : defaultSelectList }
```